### DISCUSSION TITLE
project for spdlog

### DISCUSSION THREAD

**Discussion by foxnight2** [gabime/spdlog#2953] [Q&A]
Do you know where can I find the sample project for spdlog?

**Comment by tt4g** [comment]
spdlog (v1.12.0) usage example is here: https://github.com/gabime/spdlog/tree/v1.12.0/example

**Comment by foxnight2** [comment]
Thanks for your link.

But I would like other application project .

**Reply by tt4g** [reply]
We are not aware of any projects using spdlog as they do not report it to this repository.

Search for "spdlog" in [GitHub code search](https://github.com/features/code-search) and you will find OSS that use spdlog.
